### DISCUSSION TITLE
Plan for supporting data destinations

### DISCUSSION THREAD

**Discussion by wangxiaoying** [sfu-db/connector-x#62] [Ideas]
Other than Pandas.DataFrame, we plan to support the following destinations:

- Dask.DataFrame
- Modin.DataFrame
- PyArrow

Reply here if you have other ideas!

**Comment by ritchie46** [comment]
I think that using arrow as data destination gives you a lot of future destinations for free. I believe arrow has the fastest algorithm to ingest data into pandas without triggering block manager re-allocations. This would give you zero copy transfer to products build around apache arrow like Polars and Dremio. And probably fast ingestion to other products.

I could help on the arrow interop.
